src/cmd/run.js

```diff
@@ -82,14 +82,15 @@ export function defaultReloadStrategy(
     createWatcher = defaultWatcherCreator,
   }: ReloadStrategyOptions = {}
 ): void {
-  let watcher: Watchpack;
+  const watcher: Watchpack = (
+      createWatcher({addonId, client, sourceDir, artifactsDir})
+    );
```

Comment:
This is indented too far. It should look like this:
````javascript
const watcher: Watchpack = (
  createWatcher({addonId, client, sourceDir, artifactsDir})
);
````
This provides a visual indication to the reader that the block is connected to the line opening with `const watcher...`.